scripts/openroad/common/io.tcl

```diff
@@ -0,0 +1,163 @@
+# Copyright 2022 Efabless Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $::env(SCRIPTS_DIR)/openroad/common/set_global_connections.tcl
+
+proc read_netlist {args} {
+    puts "Reading netlist..."
+
+    if {[catch {read_verilog $::env(CURRENT_NETLIST)} errmsg]} {
+        puts stderr $errmsg
+        exit 1
+    }
+
+    link_design $::env(DESIGN_NAME)
+
+    if { [info exists ::env(CURRENT_SDC)] } {
+        if {[catch {read_sdc $::env(CURRENT_SDC)}]} {
+            puts stderr $errmsg
+            exit 1
+        }
+    }
+
+}
+
+proc read_libs {args} {
+    sta::parse_key_args "read_libs" args \
+        keys {-override}\
+        flags {-multi_corner}
+
+    set libs $::env(LIB_SYNTH_COMPLETE)
+
+    if { [info exists keys(-override)] } {
+        set libs $::env(LIB_SYNTH_COMPLETE)
```

Comment:
shouldn't this be `set libs $keys(-override)`

Reply:
... yes